controllers/articles.controller.js

```diff
@@ -52,3 +56,23 @@ exports.postCommentByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (typeof inc_votes !== "number") {
+    return res
+      .status(400)
+      .send({ msg: "Bad request, inc_votes must be a number" });
+  }
+
+  updateArticleVotes(article_id, inc_votes)
+    .then(() => fetchArticleById(article_id))
+    .then((updatedArticle) => {
+      res.status(200).send({ article: updatedArticle });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Good work, good to see you returning the whole article on a key of `article`, this is good practice.